There was a problem during execution

An error occurred：
```
Worker osm_pass1_process-1 died
java.lang.ExceptionInInitializerError
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at com.onthegomap.planetiler.util.ThreadLocalTransliterator.getInstance(ThreadLocalTransliterator.java:25)
        at com.onthegomap.planetiler.util.Translations.lambda$static$0(Translations.java:22)
        at java.base/java.lang.ThreadLocal$SuppliedThreadLocal.initialValue(ThreadLocal.java:305)
        at java.base/java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:195)
        at java.base/java.lang.ThreadLocal.get(ThreadLocal.java:172)
        at com.onthegomap.planetiler.util.Translations.transliterate(Translations.java:146)
        at com.onthegomap.planetiler.util.LanguageUtils.transliteratedName(LanguageUtils.java:51)
        at org.openmaptiles.util.OmtLanguageUtils.getNames(OmtLanguageUtils.java:92)
        at org.openmaptiles.layers.Waterway.preprocessOsmRelation(Waterway.java:152)
        at com.onthegomap.planetiler.ForwardingProfile.preprocessOsmRelation(ForwardingProfile.java:98)
        at com.onthegomap.planetiler.reader.osm.OsmReader.processPass1Blocks(OsmReader.java:259)
        at com.onthegomap.planetiler.worker.WorkerPipeline$Builder.lambda$sinkTo$2(WorkerPipeline.java:275)
        at com.onthegomap.planetiler.worker.Worker.lambda$new$0(Worker.java:41)
        at com.onthegomap.planetiler.worker.Worker.lambda$new$1(Worker.java:68)
        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.MissingResourceException: Could not find the bundle com/ibm/icu/impl/data/icudt72b/translit/root
        at com.ibm.icu.impl.ResourceBundleWrapper.getBundleInstance(ResourceBundleWrapper.java:134)
        at com.ibm.icu.util.UResourceBundle.instantiateBundle(UResourceBundle.java:389)
        at com.ibm.icu.util.UResourceBundle.getBundleInstance(UResourceBundle.java:149)
        at com.ibm.icu.util.UResourceBundle.getBundleInstance(UResourceBundle.java:110)
        at com.ibm.icu.text.Transliterator.<clinit>(Transliterator.java:2172)
        ... 22 more
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:95)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
        at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)
Caused by: com.onthegomap.planetiler.util.Exceptions$FatalPlanetilerException: java.util.concurrent.ExecutionException: java.lang.ExceptionInInitializerError
        at com.onthegomap.planetiler.util.Exceptions.throwFatalException(Exceptions.java:29)
        at com.onthegomap.planetiler.stats.ProgressLoggers.await(ProgressLoggers.java:367)
        at com.onthegomap.planetiler.stats.ProgressLoggers.awaitAndLog(ProgressLoggers.java:352)
        at com.onthegomap.planetiler.reader.osm.OsmReader.pass1(OsmReader.java:221)
        at com.onthegomap.planetiler.Planetiler.lambda$addOsmSource$0(Planetiler.java:199)
        at com.onthegomap.planetiler.Planetiler.lambda$ifSourceUsed$9(Planetiler.java:841)
        at com.onthegomap.planetiler.Planetiler.run(Planetiler.java:720)
        at org.openmaptiles.OpenmaptilesApplication.run(OpenmaptilesApplication.java:55)
        at org.openmaptiles.OpenmaptilesApplication.main(OpenmaptilesApplication.java:15)
        ... 8 more
Caused by: java.util.concurrent.ExecutionException: java.lang.ExceptionInInitializerError
        at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
        at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
        at com.onthegomap.planetiler.stats.ProgressLoggers.await(ProgressLoggers.java:361)
        ... 15 more
Caused by: java.lang.ExceptionInInitializerError
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at com.onthegomap.planetiler.util.ThreadLocalTransliterator.getInstance(ThreadLocalTransliterator.java:25)
        at com.onthegomap.planetiler.util.Translations.lambda$static$0(Translations.java:22)
        at java.base/java.lang.ThreadLocal$SuppliedThreadLocal.initialValue(ThreadLocal.java:305)
        at java.base/java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:195)
        at java.base/java.lang.ThreadLocal.get(ThreadLocal.java:172)
        at com.onthegomap.planetiler.util.Translations.transliterate(Translations.java:146)
        at com.onthegomap.planetiler.util.LanguageUtils.transliteratedName(LanguageUtils.java:51)
        at org.openmaptiles.util.OmtLanguageUtils.getNames(OmtLanguageUtils.java:92)
        at org.openmaptiles.layers.Waterway.preprocessOsmRelation(Waterway.java:152)
        at com.onthegomap.planetiler.ForwardingProfile.preprocessOsmRelation(ForwardingProfile.java:98)
        at com.onthegomap.planetiler.reader.osm.OsmReader.processPass1Blocks(OsmReader.java:259)
        at com.onthegomap.planetiler.worker.WorkerPipeline$Builder.lambda$sinkTo$2(WorkerPipeline.java:275)
        at com.onthegomap.planetiler.worker.Worker.lambda$new$0(Worker.java:41)
        at com.onthegomap.planetiler.worker.Worker.lambda$new$1(Worker.java:68)
        at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.MissingResourceException: Could not find the bundle com/ibm/icu/impl/data/icudt72b/translit/root
        at com.ibm.icu.impl.ResourceBundleWrapper.getBundleInstance(ResourceBundleWrapper.java:134)
        at com.ibm.icu.util.UResourceBundle.instantiateBundle(UResourceBundle.java:389)
        at com.ibm.icu.util.UResourceBundle.getBundleInstance(UResourceBundle.java:149)
        at com.ibm.icu.util.UResourceBundle.getBundleInstance(UResourceBundle.java:110)
        at com.ibm.icu.text.Transliterator.<clinit>(Transliterator.java:2172)
        ... 22 more

```
The icu was installed according to the error that occurred, but the icu still cannot be found during the execution process !
https://github.com/unicode-org/icu source code to install ICU library
command execution result：
`icu-config --version`
`69.1`

--- a/planetiler-core/src/main/java/com/onthegomap/planetiler/util/ThreadLocalTransliterator.java
+++ b/planetiler-core/src/main/java/com/onthegomap/planetiler/util/ThreadLocalTransliterator.java
@@ -1,7 +1,11 @@
 package com.onthegomap.planetiler.util;
 
+import com.ibm.icu.text.Transliterator;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A {@link com.ibm.icu.text.Transliterator} that does not share any static data with other thread local
@@ -12,6 +16,8 @@ import java.lang.reflect.Method;
  * used across different threads in order to transliterate without contention.
  */
 public class ThreadLocalTransliterator {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ThreadLocalTransliterator.class);
+  private static final AtomicBoolean loggedOnce = new AtomicBoolean(false);
   private final ClassLoader classLoader = DuplicateClassLoader.duplicateClassesWithPrefix("com.ibm.icu");
 
   /**
@@ -31,8 +37,14 @@ public class ThreadLocalTransliterator {
           throw new IllegalStateException(e);
         }
       };
-    } catch (ClassNotFoundException | InvocationTargetException | NoSuchMethodException | IllegalAccessException e) {
-      throw new IllegalStateException(e);
+    } catch (ClassNotFoundException | InvocationTargetException | NoSuchMethodException | IllegalAccessException |
+      ExceptionInInitializerError e) {
+      if (!loggedOnce.get() && loggedOnce.compareAndSet(false, true)) {
+        LOGGER.warn("Could not get thread-local transliterator, falling back to slower shared instance: {}",
+          e.toString());
+      }
+      Transliterator t = Transliterator.getInstance(id);
+      return t::transliterate;
     }
   }
 

